api/src/pcapi/core/providers/titelive_apply_new_filters_on_history.py

```diff
@@ -0,0 +1,346 @@
+import argparse
```

Comment:
Ce script est trop long, nouveau plan avec optimisation : 

- script 1 : sortir un fichier CSV avec les product id des produits inéligibles -> approx 215.763 lignes 
- script 2 : rejeter les offres + désindexer et sortir les `offerIds` rejetées dans un CSV 
- script 3 : annuler les réservations depuis les `offerIds`
  - ne plus appeler les fonctions suivante :  
    - `amplitude_events.track_cancel_booking_event`
    - `update_external_user`
    - `update_external_pro`
    - `search.async_index_offer_ids`
  - optimiser `_cancel_bookings_from_stock` -> regarder que les réservations qui sont a l'état confirmed  sur le stock de l'offre
- script 4: delete des favoris depuis les offerIds